common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -1070,14 +1069,44 @@ div.problem {
     &.warning {
       @include notification-by-type($warning-color);
     }
+    &.hint {
+      border: 1px solid $uxpl-gray-background;
+      border-radius: 7px;
```

Comment:
To achieve the vertical alignment, change line 1058 padding
padding: ($baseline * 0.4) ($baseline / 2) ($baseline / 5);

and add line-height: 1.5em;   (tested ok in Studio to use em here)